src/XrdDig/XrdDigAuth.cc

```diff
@@ -233,10 +233,6 @@ bool XrdDigAuth::Parse(XrdOucStream &aFile, int lNum)
    int  k, n, bLeft = sizeof(buff);
    bool aOK = false, tfVal;
 
-// Clear the arec the fastest way possible
-//
-   memset(aEnt.eP, 0, sizeof(XrdDigAuthEnt));
-
```

Comment:
Uh, this doesn't look kosher. You just eliminated initializing aEnt so I doubt his will work.

Reply:
Again, the compiler was complaining that an object with a non-trivial constructor is being initialized with memset. 

Now the fix rational:

aEnt.eP is a newly created instance of 'XrdDigAuthEnt':
https://github.com/xrootd/xrootd/blob/fedora-rawhide-fix/src/XrdDig/XrdDigAuth.cc#L227

note that I modified the default constructor of 'XrdDigAuthEnt' :
https://github.com/xrootd/xrootd/commit/60da79db805d122a7190535b8dead0f306e5a2f9#diff-a726e008bf733c6a23ce19da88801122R56

now, it initializes all the members to 0, and clears member arrays, so it is equivalent to the memset I removed.

Reply:
OK, got it.